tests/common/test_logger.py

```diff
@@ -376,15 +390,15 @@ def test_telemetry_logger_check_all_file_logs_written_when_events_gt_1000(self,
         exception_caught = False
         prefix = "YoloLogger"
 
-        lg = logger.Logger(logger.DEFAULT_LOGGER, prefix)
-        lg.add_appender(logger.AppenderType.FILE, logger.LogLevel.INFO, path=self.log_file)
-        lg.add_appender(logger.AppenderType.TELEMETRY, logger.LogLevel.WARNING, path=add_log_event)
+        logger.add_logger_appender(logger.AppenderType.FILE, logger.LogLevel.INFO, path=self.log_file)
```

Comment:
NIT: The current test name is `test_telemetry_logger_check_all_file_logs_written_when_events_gt_**1000**` Could you please rename it to something like `*_gt_MAX_EVENTS` so that incase we change the size later we dont need to modify this test case again. Also in Line 406 we're checking explicitly for 1000, can you change it to the new constant you added for MAX_EVENTS.
Thanks!